### ISSUE TITLE
comma missed at manualcomma missed

### ISSUE THREAD

**Issue by koanvi** [sequelize/sequelize#12748]
## Issue Description

comma missed at manual page
https://sequelize.org/master/manual/model-querying-basics.html

### What was unclear/insufficient/not covered in the documentation

Model.findAll({
  attributes: [
    'foo',
    [sequelize.fn('COUNT', sequelize.col('hats')), 'n_hats']
    'bar'
  ]
});

### If possible: Provide some suggestion on how we can enhance the docs
Model.findAll({
  attributes: [
    'foo',
    [sequelize.fn('COUNT', sequelize.col('hats')), 'n_hats'],
    'bar'
  ]
});

**Comment by arl-ray** [comment]
```
Model.findAll({
attributes: [
    'foo',
    [sequelize.fn('COUNT', sequelize.col('hats')), 'n_hats'] // missing comma here
    'bar'
  ]
});
```

**Comment by papb** [comment]
Closed by #12786
